turbinia/lib/recipe_helpers.py

```diff
@@ -166,4 +170,44 @@ def validate_recipe(recipe_dict):
         return (False, message)
       tasks_with_recipe.append(recipe_item)
 
-  return (True, '')
\ No newline at end of file
+  return (True, '')
+
+
+def validate_recipe_conditions(args):
+  """Validates necessary pre-conditions to use a recipe.
+
+  Args:
+    args(namespace): turbiniactl args.
+  
+  Raises:
+    TurbiniaException: if the conditions are not met.
+  """
+  if args.recipe and args.recipe_path:
```

Comment:
Maybe we can add a `try` for `AttributeError` (if that's the right error) in case a different object without these attributes gets passed in/

Reply:
Done